--- efax-0.9/fax.orig	Mon May  6 17:46:23 2002
+++ efax-0.9/fax	Mon May  6 17:48:34 2002
@@ -424,7 +424,8 @@
 		fi
 		case $f in
 		*.[0-9][0-9][0-9]) FILES="$FILES $f" ;;	# skip image files
-	   	*)	if echo ${f}.001: $f \; x | make -r -q -f - ; then
+#	   	*)	if echo ${f}.001: $f \; x | make -r -q -f - ; then
+	   	*)	if [ "$f.001" -nt "$f" ]; then
 				echo ${f}.nnn is up-to-date
 			else
 	 			$RM ${f}.[0-9][0-9][0-9]
